src/nft/components/collection/Card.tsx

```diff
@@ -583,89 +652,95 @@ const ProfileNftDetails = ({ asset, hideDetails }: ProfileNftDetailsProps) => {
   const shouldShowUserListedPrice = !asset.notForSale && asset.asset_contract.tokenType !== NftStandard.Erc1155
 
   return (
-    <Box overflow="hidden" width="full" flexWrap="nowrap">
+    <PrimaryRowContainer>
       <PrimaryRow>
         <PrimaryDetails>
-          <TruncatedTextRow className={bodySmall} style={{ color: themeVars.colors.textSecondary }}>
+          <TruncatedTextRow color="textSecondary">
             {!!asset.asset_contract.name && <span>{asset.asset_contract.name}</span>}
           </TruncatedTextRow>
           {asset.collectionIsVerified && <VerifiedIcon height="18px" width="18px" />}
         </PrimaryDetails>
         {!hideDetails && <DetailsLink />}
       </PrimaryRow>
-      <Row justifyItems="flex-start">
-        <TruncatedTextRow
-          className={body}
-          style={{
-            color: themeVars.colors.textPrimary,
-          }}
-        >
-          {assetName()}
-        </TruncatedTextRow>
+      <ProfileAssetNameRow>
+        <TruncatedTextRow color="textPrimary">{assetName()}</TruncatedTextRow>
         {asset.susFlag && <Suspicious />}
-      </Row>
-      <TruncatedTextRow className={buttonTextMedium} style={{ color: themeVars.colors.textPrimary }}>
+      </ProfileAssetNameRow>
+      <FloorPriceRow>
```

Comment:
You could remove FloorPriceRow and use `ThemedText.Subheader` with a lineheight of 20px

Reply:
it actually extends another styled component